### PR TITLE
Support guatemala currency

### PR DESCRIPTION
**Pull request by monroy95**
Mario Monroy m.monroyc22@gmail.com

### Changes proposed in this pull request:
```
    modified:   num2words/__init__.py
    modified:   README.rst
    new file:   num2words/lang_ES_GT.py
    new file:   tests/test_es_gt.py
```
### Status

- [x] READY
- [ ] HOLD
- [ ] WIP (Work-In-Progress)

### How to verify this change
python bin/num2words 1 -l es_GT --to currency
un quetzal con cero centavos

python bin/num2words 1.02 -l es_GT --to currency
un quetzal con dos centavos

python bin/num2words 2.02 -l es_GT --to currency
dos quetzales con dos centavos

python bin/num2words 1.01 -l es_GT --to currency
un quetzal con un centavo

### CODE REVIEW

**Comment by monroy95** [comment]
> @monroy95 thanks a lot for taking the time to contribute with this project.
> 
> In order to avoid a lot of code duplication, we introduced on the past the **CURRENCY_FORMS**.
> We have not refactored all the coded to remove the duplication but we want to avoid creating new ones. If you want to add the Guatemala currency, then you should be able to do by adding it to the existing **CURRENCY_FORMS** on the file `num2words/lang_ES.py`
> 
> Then it can be used like this:
> 
> ```
> >>> from num2words import num2words
> >>> num2words(34, to='currency', lang='es', currency='ESP')
> 'cero pesetas con treinta y cuatro céntimos'
> >>> num2words(34, to='currency', lang='es', currency='GTQ')
> 'cero quetzales con treinta y cuatro centavos'
> ```
> 
> You could add new tests on `tests/test_es.py` to cover the use of this currency.
> Thanks a lot again.

Thanks, change added.

preview: 
Captura de Pantalla 2019-12-27 a la(s) 17 04 55

File: num2words/lang_ES_GT.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# MA 02110-1301 USA
+
+from __future__ import print_function, unicode_literals
+
+from .lang_ES import Num2Word_ES
+
+
+class Num2Word_ES_GT(Num2Word_ES):
+    CURRENCY_FORMS = {
+        'GTQ': (('quetzal', 'quetzales'), ('centavo', 'centavos')),
```
[the remark this answers is not in the document]
Comment (monroy95) [reply num2words/lang_ES_GT.py]: Thank you very much for the feedback, you are right, suggested change added

File: README.rst [hunk 2]
```diff
@@ -84,6 +84,7 @@ Besides the numerical argument, there are two main optional arguments.
 * ``en_IN`` (English - India)
 * ``es`` (Spanish)
 * ``es_CO`` (Spanish - Colombia)
+* ``es_GT`` (Spanish - Guatemala)
```
Comment (erozqba) [inline README.rst]: @monroy95 then we don't need this change.

Comment (monroy95) [reply README.rst]: Added suggested change

**Comment by erozqba** [comment]
@monroy95 Could you rebase your branch? I don't understand why coveralls it's failing and github will not let me to merge. :sob:

**Comment by monroy95** [comment]
> @monroy95 Could you rebase your branch? I don't understand why coveralls it's failing and github will not let me to merge. 😭

No problem, I'll try again 😄

**Pull request closed without merging**